src/content/en/2020/caching.md

```diff
@@ -15,11 +15,11 @@ discuss: 2056
 results: https://docs.google.com/spreadsheets/d/1fYmpSN3diOiFrscS75NsjfsrKXzxxhUMNcYSqXnQJQU/
 queries: 20_Caching
 #featured_quote: Caching benefits both the end users (they get their web pages quickly) and the companies serving the web pages (reducing the load on their servers). Caching really is a win-win!
-#featured_stat_1: TODO
+#featured_stat_1: **One quarter** of all HTTP responses don't include any caching information, which means they're probably being cached wrongly
 #featured_stat_label_1: TODO
-#featured_stat_2: TODO
+#featured_stat_2: More than **one fifth** of all responses aren't using revalidation, resulting in wasted bandwidth and increased latency
 #featured_stat_label_2: TODO
-#featured_stat_3: TODO
+#featured_stat_3: Our chapter has a cat picture!
```

Comment:
😸 Can we use a real stat here instead?

Reply:
We could. I was trying to add some humor :) I'll dig out a useful one...

Reply:
@rviscomi what should go in the `featured_stat_label` fields?

Reply:
Here's an example of how it looks on the home page

![image](https://user-images.githubusercontent.com/1120896/101710365-522e4400-3a5f-11eb-9000-dab2ee8d0359.png)

The big text like 72.58% is the stat and "percent of length values..." is the label.

Reply:
Ah - ok then, I'll change it - currently I have the `stat` just the text string. More commits!

Reply:
Also, it looks like there's an error trying to compress at least one of the images - probably the new `luna-cat.png` image... Any thoughts on what I should do? I can make it smaller, if that helps - right now it's about 2MB.